backend/api/v1/endpoints/auth.py

```
@@ -23,10 +31,13 @@
 # Router
 router = APIRouter()
 
-
+allowed_keywords = ["stud", "edu", "university", "college", "ac", "edu", "institution"]
 # ✅ Signup Route
 @router.post("/signup/")
 async def signup(user: UserCreate, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `signup`

(ANN201)